examples/todomvc/tests/create/add-todo-with-special-characters.spec.ts

```diff
@@ -0,0 +1,34 @@
+// spec: specs/basic-operations.md
+// seed: tests/seed.spec.ts
+
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://demo.playwright.dev/todomvc');
```

Comment:
This looks very weird in each file. Update the prompt to include it into each test instead?

Reply:
That's a good point, we need to figure out how reuse should work.